cmd/snap/notes.go

```diff
@@ -56,6 +57,10 @@ func (n *Notes) String() string {
 		ns = append(ns, "try")
 	}
 
+	if n.Inactive {
+		ns = append(ns, "inactive")
```

Comment:
`n.Disabled` and `disabled`, so we're using the terminology that was used to put it in that state. We should likely s/Active/Enabled/ at some point, to align internal and external terminology.